### PR TITLE
Tweaks space syndie listening post and lava syndie base.

### PR DESCRIPTION
**Pull request by WJohn**
Both bases now have their broadcast/listening channels turned into 1 intercom, which starts with listening on, broadcasting off (so you don't accidentally alert the station while talking on your radio near them). Space listening post now has a real microwave instead of a model, a library book management console, and the rocks surrounding it can now contain minerals.

:cl: WJohnston
fix: Space syndie listening post now has a real microwave, and a book management console. Lava listening post comms room intercom now no longer starts with broadcast on.
add: the syndicate drop ship and fighter pods from the caravan ambush can now be flown to the syndicate listening post. The fighter pods also now have xray cameras because regular ones were super useless.
/:cl:

### CODE REVIEW

**Comment by Kromgar** [comment]
Who was the dick who put them to be on right when you started?

Oh god I screamed internally when I saw my message about framing the captain got broadcasted to the whole crew

**Comment by Ratvar** [comment]
Two intercoms let you say false messages to many radiochanels at once, leave second intercom in

**Comment by WJohn** [comment]
That's how they always were in the original design, I think it's because they were ripped from ye olde DJ station out in space. So whoever made that, got copied by okand, got copied by me. I'm only fixing it all these years later.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 1]
```diff
@@ -5798,11 +5791,11 @@
 "oj" = (
 /obj/structure/table/reinforced,
 /obj/item/device/radio/intercom{
-	broadcasting = 1;
+	broadcasting = 0;
 	dir = 8;
 	freerange = 1;
-	listening = 0;
-	name = "Pirate Radio Broadcast Channel"
+	listening = 1;
```
Comment (ShizCalev) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: DV

File: _maps/RandomRuins/SpaceRuins/listeningstation.dmm [hunk 2]
```diff
@@ -53,6 +51,17 @@
...
 	},
 /obj/effect/decal/cleanable/dirt,
+/obj/machinery/newscaster{
+	pixel_y = 32
+	},
+/obj/item/device/radio/intercom{
+	broadcasting = 0;
+	dir = 8;
+	freerange = 1;
+	listening = 1;
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/listeningstation.dmm]: DV

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**